CollaberativeFilteredSimilarUsersGenerator.py

```diff
@@ -0,0 +1,15 @@
+
+from src.recommendation_system.recommendation_flow.candidate_generators.AbstractGenerator import AbstractGenerator
+from src.data_structures.user_based_recommender.golf.UserBasedRecommender import UserBasedRecommender
+
+
+class CollaberativeFilteredSimilarUsersGenerator(AbstractGenerator):
+    def _get_content_ids(self, user_id, limit, offset, seed, starting_point):
+
+        recommender = UserBasedRecommender()
```

Comment:
where is the implementation for this?

Reply:
uploaded this file just now [UserBasedRecommender.py]